nservicebus/upgrades/7to8/dependency-injection.md

```diff
@@ -21,6 +21,13 @@ The following adapter packages will no longer be provided:
 
 Instead of the container adapter packages, use the [NServiceBus.Extensions.Hosting](/nservicebus/hosting/extensions-hosting.md) package or the [externally managed container mode](/nservicebus/dependency-injection/#externally-managed-mode) to use a third party dependency injection container. See the [migrating to the Generic Host](#microsoft-generic-host) or [migrating to externally managed mode](#externally-managed-container-mode) sections for further information.
 
+## Behavior changes
```

Comment:
I tried to keep it as short as possible. If you think it would be good to have samples let me know. I felt it would be overkill